### PR TITLE
Update FAQ.md

### LINKED ISSUE #870: Configuring processing of deleted users
How long does it take for users in the section "User waiting for permanent deletion" to be deleted? Is this a setting that can be configured?

### PR DESCRIPTION
**Pull request by AndyHee**
Update documentation about deletion process.

This closes #870

### CODE REVIEW

File: doc/FAQ.md [hunk 1]
```diff
@@ -126,16 +126,11 @@ A **hidden contact** will not be displayed in any "friend list" (except to you).
 However a hidden contact will appear normally in conversations and this may expose his/her hidden status to anybody who can see the conversation.
 
 <a name="removed"></a>
-### What happens when an account is removed? Is it truly deleted?
+### What happens when an account is removed?
 
-If you delete your account, we will immediately remove all your content on **your** server.
+If you delete your account, it will be scheduled for permanent deletion in *seven days*. As soon as you activate the deletion process you won't be able to logon any more. Only the administrator of your node can halt this process prior to permanent deletion.
```
Comment (MrPetovan) [inline doc/FAQ.md]: `login`

File: doc/FAQ.md [hunk 2]
```diff
@@ -126,16 +126,11 @@ A **hidden contact** will not be displayed in any "friend list" (except to you).
...
 
-Then Friendica issues requests to all your contacts to remove you.
-This will also remove you from the global directory.
-Doing this requires your account and profile still to be "partially" available for up to 24 hours in order to establish contact with all your friends.
-We can block it in several ways so that it appears empty and all profile information erased, but will then wait for 24 hours (or after all of your contacts have been notified) before we can physically remove it.
-
-After that, your account is deleted.
+After the elapsed time of seven days, all your posts, messages, photos, and personal information stored on your node will be deleted. Your node will also issue removal requests to all your contacts; this will also remove your profile from the global directory if you are listed. Your username cannot be reissued for future sign-ups for security reasons.
```
Comment (MrPetovan) [inline doc/FAQ.md]: Formatting: Please add a line feed after periods.

File: doc/FAQ.md (hunk 1 above)
Comment (MrPetovan) [inline doc/FAQ.md]: Formatting: Please add a line feed after periods.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by AndyHee** [comment]
Had forgotten about the line breaks. All done now.

**Review by MrPetovan — APPROVED** [review]

File: doc/FAQ.md [hunk 3]
```diff
@@ -126,16 +126,15 @@ A **hidden contact** will not be displayed in any "friend list" (except to you).
 However a hidden contact will appear normally in conversations and this may expose his/her hidden status to anybody who can see the conversation.
 
 <a name="removed"></a>
-### What happens when an account is removed? Is it truly deleted?
+### What happens when an account is removed?
 
-If you delete your account, we will immediately remove all your content on **your** server.
+If you remove your account, it will be scheduled for permanent deletion in *seven days*. 
+As soon as you activate the deletion process you won't be able to login any more. 
+Only the administrator of your node can halt this process prior to permanent deletion.
```
Comment (tobiasd) [inline doc/FAQ.md]: How? Is it documented anywhere for the admin how to stop the deletion?

**Pull request merged**